app/models/manageiq/providers/vmware/infra_manager/vim_connect_mixin.rb

```diff
@@ -20,7 +20,13 @@ def vim_connect(options = {})
 
     Thread.current[:miq_vim][conn_key] ||= begin
       require 'VMwareWebService/MiqVim'
-      MiqVim.new(*options.values_at(:ip, :user, :pass, :cache_scope, :monitor_updates, :pre_load))
```

Comment:
what happened to the `monitor_updates` parameter?

Reply:
:+1: missed that one thanks